hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/QueryResultProducerImpl.java

```diff
@@ -59,8 +55,9 @@ public class QueryResultProducerImpl implements QueryResultProducer {
      * batch jobs where low streaming latency isn't required, but rather we
      * want to return full pages of results to the client.
      */
-    public QueryResultProducerImpl(boolean blockForNextItem) {
-        this.blockForNextItem = blockForNextItem;
+    public QueryResultProducerImpl(boolean isBatch) {
```

Comment:
What about jobs like this:
```
INSERT INTO a
SELECT * FROM b LIMIT 100;
```
Such job would not use `QueryResultProducer`. I'm not sure we support it now, but we may support it some day.

And the javadoc is stale.